resources/js/Pages/Admin/Rooms/RoomsList.vue

```diff
@@ -1,275 +0,0 @@
-                        <div class="grid grid-cols-7">
-                            <div class="text-md mx-3">Room Name</div>
-                            <div class="text-md mx-3">Room Number</div>
-                            <div class="text-md mx-2">Floor Number</div>
-                            <div class="text-md mx-3">Building</div>
-                            <div class="text-md mx-3">Status</div>
-                            <div class="text-md mx-3">Room Type</div>
-                        </div>
-                    
-                        <div v-for="room in rooms" :key="room.id" class="grid flex items-center">
-                            <div class="grid grid-cols-7">
-                                <div class="text-md mx-3">
-                                    {{ room.name }}
-                                </div>
-                                <div class="text-md mx-3">
-                                    {{ room.number }}
-                                </div>
-                                <div class="text-md mx-2">
-                                    {{ room.floor }}
-                                </div>
-                                <div class="text-md mx-3">
-                                    {{ room.building }}
-                                </div>
-                                <div class="text-md mx-3">
-                                    {{ room.status }}
-                                </div>
-                                <div class="text-md mx-3">
-                                    {{ room.room_type }}
-                                </div>
-                                <div class="text-md mx-3">
-                                <jet-dropdown align="right" width="48">
-                                   <template #trigger>
-                                        <button
-                                        class="flex items-center text-sm font-medium text-gray-500 hover:text-gray-700 hover:border-gray-300 focus:outline-none focus:text-gray-700 focus:border-gray-300 transition duration-150 ease-in-out"
-                                        >
-                                        <div>. . .</div>
-                                    </button>
-                                   </template>     
-
-                                   <template #content> 
-                                    <div class="text-md mx-3">
-                                    <button
-                                        class="cursor-pointer ml-6 text-sm text-blue-800 focus:outline-none"
-                                        @click="roomBeingUpdated = room"
-                                    >
-                                        Update
-                                    </button>
-                                </div>
-                                <div class="text-md mx-3">
-                                    <jet-dropdown-link :href="'/rooms/'+room.id+'/blackouts'">
```

Comment:
This table is currently missing the function to blackout rooms. The old RoomList page had a button in its dropdown leading admins to all active blackouts for the selected room, and allowed admins to create a new blackout. 

The new table should keep that, and hopefully could also put the 4 action button columns into one column as a dropdown to keep the list more compact.

Reply:
I just saw that this was missing, I will be turning the last columns of the table into a dropdown, thank Alex!